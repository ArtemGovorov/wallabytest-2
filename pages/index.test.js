import React from 'react';
import toJson from 'enzyme-to-json';
import { shallow } from 'enzyme';
import IndexPage from 'pages/index';

describe('<IndexPage />', () => {
  const component = shallow(<IndexPage bar="hello world" />);
  test('renders as expected', () => {
    const tree = toJson(component);
    expect(tree).toMatchSnapshot();
  });
});
