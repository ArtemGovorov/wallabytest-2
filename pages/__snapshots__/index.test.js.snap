// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<IndexPage /> renders as expected 1`] = `
<pre>
   
  hello world
   text: hi there :)
</pre>
`;
